### PR TITLE
post comments

### PR DESCRIPTION
**Pull request by Krishnainani**
This pull request will add the post comments section

### CODE REVIEW

File: src/components/PostComments.jsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  const handleName = (event) => {
+    setName(event.target.value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    postComments(name, body, id).then(({ comment }) => {
+      setBody("");
+      setName("tickle122");
+      return setNewComment([comment]);
```
Comment (douglashellowell) [inline src/components/PostComments.jsx]: why are you setting newComment to an array here?

It's only ever going to be one comment?

File: src/components/PostComments.jsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                  {comment.body}
+                  <h4>By: {comment.author}</h4>
+                  {date.toDateString()} {date.toLocaleTimeString()}
+                  <br />
+                  <br />
+                  <button type="display">Likes: {comment.votes}</button>
+                </section>
+              </li>
+            );
+          })}
```
Comment (douglashellowell) [inline src/components/PostComments.jsx]: // having the new comment appear both here *and* in the list is a little confusing 🥴

Ignore this!

I think i was mistaken, apologies!

**Comment by douglashellowell** [comment]
---

posting a comment with the comments hidden gives no feedback that it's worked!

---

The buttons named "Post Comment" and "submit" both sound like they do the same thing!

---

(paused here due to infinite loop)

File: src/components/PostComments.jsx (hunk 1 above)
Comment (Krishnainani) [reply src/components/PostComments.jsx]: while rendering it is showing the titles and CSS before posting the comment so have done that so I can use map and make it appear when I click submit

File: src/components/PostComments.jsx (hunk 2 above)
Comment (Krishnainani) [reply src/components/PostComments.jsx]: just an indication to the user that an comment is posted and the list is only viewed when we click show comments

File: src/components/PostComments.jsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+              <li className="cards" key={comment.comment_id}>
+                <section id="card">
+                  {comment.body}
+                  <h4>By: {comment.author}</h4>
+                  {date.toDateString()} {date.toLocaleTimeString()}
+                  <br />
+                  <br />
+                  <button type="display">Likes: {comment.votes}</button>
+                </section>
+              </li>
```
Comment (douglashellowell) [inline src/components/PostComments.jsx]: if you end up repeating UI like this... don't copy paste - make a component!

Comment (Krishnainani) [reply src/components/PostComments.jsx]: ok will refactor

**Comment by douglashellowell** [comment]
Nicely done!

There are a few confusing behaviours that your users might run into using your comments posting component

As mentioned there's no feedback if the comments are hidden
if a user posts multiple comments they can no longer see their previous ones
On a slow network connection the user might not see anything happen for a while - you should give the user some immediate feedback that the post request is happening

After you've added some feedback on button click this will be ready to merge 💪🏻

**Pull request merged**